docs/Displaying a User Interface/Template Custom Buttons/index.md

```diff
@@ -172,8 +188,21 @@ sdlManager.screenManager.endUpdates { (error) in
 
 @![android, javaSE, javaEE]
 ```java
-SoftButtonState imageState = new SoftButtonState("<#State Name#>", null, sdlArtwork);
-SoftButtonObject softButtonObject = new SoftButtonObject("softButtonObject", Collections.singletonList(imageState), imageState.getName(), new SoftButtonObject.OnEventListener() {
+SoftButtonState imageState1 = new SoftButtonState("<#State Name1#>", null, sdlArtwork1);
+SoftButtonState imageState2 = new SoftButtonState("<#State Name2#>", null, sdlArtwork2);
+SoftButtonObject softButtonObject1 = new SoftButtonObject("softButtonObject1", Arrays.asList(imageState1, imageState2), imageState1.getName(), new SoftButtonObject.OnEventListener() {
+    @Override
+    public void onPress(SoftButtonObject softButtonObject, OnButtonPress onButtonPress) {
+    }
+
+    @Override
+    public void onEvent(SoftButtonObject softButtonObject, OnButtonEvent onButtonEvent) {
+    }
+});
+
+SoftButtonState imageState3 = new SoftButtonState("<#State Name3#>", null, sdlArtwork3);
+SoftButtonState imageState4 = new SoftButtonState("<#State Name4#>", null, sdlArtwork4);
+SoftButtonObject softButtonObject2 = new SoftButtonObject("softButtonObject2", Arrays.asList(imageState3, imageState4), imageState3.getName(), new SoftButtonObject.OnEventListener() {
```

Comment:
Maybe we should have one of the SoftButtonObjects in each example have only one state? Just to have a mix